hyf-core-values.md

```diff
@@ -2,22 +2,23 @@
 
 ## Learning
 * Our program is about problem solving and learning how to learn.
-* Take responsibility for your own learning process.
-* Tell us and your fellow students what works for you and what doesn't when it comes to learning.
-* Be active and communicate about things that are not clear.
-* Help us and our teachers improve by giving regular feedback.
```

Comment:
this one is important for me I rather keep it

Reply:
which one of the 3?

Reply:
Actually all three. You could say that they are ways/examples of taking responsibility of your own learning proces. I think it is good to give some examples on what it is that we expect if we say something like that, since it is a bit generic.